trogon/widgets/command_tree.py

```diff
@@ -5,19 +5,18 @@
 from textual.widgets import Tree
 from textual.widgets._tree import TreeNode, TreeDataType
 
-from trogon.introspect import CommandSchema, CommandName
+from trogon.schemas import CommandName, CommandSchema
 
 
 class CommandTree(Tree[CommandSchema]):
     COMPONENT_CLASSES = {"group"}
 
-    def __init__(self, label: TextType, cli_metadata: dict[CommandName, CommandSchema], command_name: str):
+    def __init__(self, label: TextType, cli_metadata: dict[CommandName, CommandSchema]):
```

Comment:
The exclusion of a specific command-name can be handled in each parsers "introspector". See:

https://github.com/Textualize/trogon/pull/38/files#diff-52fb33fe4cb2ee6540c202d94a3c5a5729af562b7e0a308e6ff92e7e9082157bR115